udecode/plate
# [Release] Version packages

This PR was opened by the [Changesets release](https://github.com/changesets/action) GitHub action. When you're ready to do a release, you can merge this and the packages will be published to npm automatically. If you're not ready to do a release yet, that's fine, whenever you add more changesets to main, this PR will be updated.


# Releases
## @udecode/plate-comments@27.0.0

### Major Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) –
    -   Migrate store to jotai@2
    -   Revert the breaking changes to `@udecode/plate-comments` made in 26.0.0

## @udecode/plate-core@27.0.0

### Major Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) –
    -   Migrate store from `jotai@1` to `jotai@2`
        -   New dependency: `jotai-x`. See <https://github.com/udecode/jotai-x>
        -   Accessing a store without an explicit provider component is no longer supported. Attempting to do so will result in a warning in the console: `Tried to access jotai store '${storeName}' outside of a matching provider.`
    -   Upgraded from `zustand@3` to `zustand@4`
        -   See <https://github.com/udecode/zustand-x>
    -   Rename `zustand-x` exports
        -   `StateActions` -> `ZustandStateActions`
        -   `StoreApi` -> `ZustandStoreApi`
        -   `createStore` -> `createZustandStore`
        -   Note that these exports are deprecated and should not be used in new code. They may be removed in a future version of Plate.
    -   `renderAboveEditable` and `renderAboveSlate`
        -   The given component is now rendered using JSX syntax, separately from the parent component. Previously, the component was called as a function, which affected how hooks were handled by React.
    -   `withHOC`
        -   Add support for `ref` prop, which is forwarded to the inner component
        -   Add `hocRef` argument, which is forwarded to the `HOC`
        -   Strengthen the type of `hocProps`

## @udecode/plate-resizable@27.0.0

### Major Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) –
    -   Migrate store to jotai@2
    -   Resizable components must now be wrapped inside a `ResizableProvider`

## @udecode/plate-caption@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-cloud@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-combobox@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-dnd@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-link@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-media@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-selection@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-suggestion@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) –
    -   Migrate store to jotai@2

## @udecode/plate-table@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) –
    -   Migrate store to jotai@2
    -   Render `TableProvider` above editable

## @udecode/plate-yjs@27.0.0

### Patch Changes

-   [#2763](https://github.com/udecode/plate/pull/2763) by [@12joan](https://github.com/12joan) – Update Zustood imports

## @udecode/plate-alignment@27.0.0



## @udecode/plate-autoformat@27.0.0



## @udecode/plate-basic-elements@27.0.0



## @udecode/plate-basic-marks@27.0.0



## @udecode/plate-block-quote@27.0.0



## @udecode/plate-break@27.0.0



## @udecode/plate-code-block@27.0.0



## @udecode/plate-common@27.0.0



## @udecode/plate-cursor@27.0.0



## @udecode/plate-emoji@27.0.0



## @udecode/plate-excalidraw@27.0.0



## @udecode/plate-find-replace@27.0.0



## @udecode/plate-floating@27.0.0



## @udecode/plate-font@27.0.0



## @udecode/plate-heading@27.0.0



## @udecode/plate-highlight@27.0.0



## @udecode/plate-horizontal-rule@27.0.0



## @udecode/plate-indent@27.0.0



## @udecode/plate-indent-list@27.0.0



## @udecode/plate-juice@27.0.0



## @udecode/plate-kbd@27.0.0



## @udecode/plate-line-height@27.0.0



## @udecode/plate-list@27.0.0



## @udecode/plate-mention@27.0.0



## @udecode/plate-node-id@27.0.0



## @udecode/plate-normalizers@27.0.0



## @udecode/plate-paragraph@27.0.0



## @udecode/plate@27.0.0



## @udecode/plate-utils@27.0.0



## @udecode/plate-reset-node@27.0.0



## @udecode/plate-select@27.0.0



## @udecode/plate-serializer-csv@27.0.0



## @udecode/plate-serializer-docx@27.0.0



## @udecode/plate-serializer-html@27.0.0



## @udecode/plate-serializer-md@27.0.0



## @udecode/plate-tabbable@27.0.0



## @udecode/plate-trailing-block@27.0.0
